sci-physics/pythia/pythia-8.3.09.ebuild

```diff
@@ -0,0 +1,188 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+MV=$(ver_cut 1-2)
+MY_P="${PN}${PV//./}"
+LHA_VER="6.2.1"
+
+DESCRIPTION="Lund Monte Carlo high-energy physics event generator"
+HOMEPAGE="https://pythia.org/"
+SRC_URI="https://pythia.org/download/${PN}${MV//./}/${MY_P}.tgz
+	test? ( lhapdf? (
+		https://www.hepforge.org/archive/lhapdf/pdfsets/v6.backup/${LHA_VER}/CT10.tar.gz
+		https://www.hepforge.org/archive/lhapdf/pdfsets/v6.backup/${LHA_VER}/MRST2007lomod.tar.gz
+		https://www.hepforge.org/archive/lhapdf/pdfsets/v6.backup/${LHA_VER}/NNPDF23_nlo_as_0119_qed_mc.tar.gz
+		https://www.hepforge.org/archive/lhapdf/pdfsets/v6.backup/${LHA_VER}/NNPDF23_nnlo_as_0119_qed_mc.tar.gz
+		https://www.hepforge.org/archive/lhapdf/pdfsets/v6.backup/${LHA_VER}/cteq66.tar.gz
+		https://www.hepforge.org/archive/lhapdf/pdfsets/v6.backup/${LHA_VER}/cteq6l1.tar.gz
+		https://www.hepforge.org/archive/lhapdf/pdfsets/v6.backup/${LHA_VER}/unvalidated/MRST2004qed.tar.gz
+	) )"
+
+SLOT="8"
+LICENSE="GPL-2"
+KEYWORDS="~amd64"
+IUSE="doc examples fastjet +hepmc3 hepmc2 lhapdf root test zlib"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="
+	^^ ( hepmc3 hepmc2 )
```

Comment:
Shouldn't this be `??` instead of `^^`? I.e. 'zero or one' instead of 'exactly one'.